apps/rush-lib/src/logic/LookupByPath.ts

```diff
@@ -22,12 +22,14 @@ interface IPathTreeNode<TItem> {
  * It is optimized for efficiently locating the nearest ancestor path with an associated value.
  *
  * @example
+ * ```ts
```

Comment:
Should the other examples also say `ts`?

Reply:
Fixed